.ci/README.md

```diff
@@ -116,37 +116,36 @@ based on those labels. Possible labels are listed below:
 ### Running the CI locally
 
 The CI can run on a local VM using the following instructions:
-- Setup a Fedora VM
-  - Create a Fedora VM having at least 4G RAM and a 30GB disk.
+- Setup a Fedora/CentOS/CentOS-Stream VM
```

Comment:
Is it smart to add CentOS knowing it's going the way of the dodo?

Reply:
While you are editing this, why not add the versions that are "known to work"?

Reply:
will do